woocommerce/woocommerce-android
# Product editing banner fix

Fixes #2409.

| Editing enabled | Editing disabled |
|---|---|
| ![image](https://user-images.githubusercontent.com/1522856/81673992-cadb0c80-944c-11ea-85ab-3babd0ce4513.png) | ![image](https://user-images.githubusercontent.com/1522856/81674034-de867300-944c-11ea-8029-ddacb2a8bdb6.png) |

To test:
1. Make sure the product editing feature is disabled in the settings
2. Go to product list
3. Notice the banner says **Limited editing available** with the right message
4. Go to settings and enable the product editing beta feature
5. Go to product list
6. Notice the banner says **New editing options available** with the right message
7. Make sure the banner is expanded
8. Kill the app
9. Open the app again
10. Go to product list
11. Notice the banner is **not** expanded
